gradle/dependencies.gradle

```diff
@@ -102,20 +104,25 @@ ext {
             commonsIO               : "commons-io:commons-io:${version.commonsIO}",
             robolectric             : "org.robolectric:robolectric:${version.robolectric}",
             json                    : "org.json:json:${version.jsonVersion}",
+            androidxTestKtx         : "androidx.test:core-ktx:${version.androidxTestKtx}",
+            mockk                   : "io.mockk:mockk:${version.mockk}",
 
             // play services
             gmsLocation             : "com.google.android.gms:play-services-location:${version.gmsLocation}",
-            errorprone              : "com.google.errorprone:error_prone_core:${version.errorprone}"
+            errorprone              : "com.google.errorprone:error_prone_core:${version.errorprone}",
+
+            kotlinxSerializationJson: "org.jetbrains.kotlinx:kotlinx-serialization-json:1.7.3"
     ]
 
     pluginDependencies = [
-            gradle              : "com.android.tools.build:gradle:${pluginVersion.gradle}",
-            checkstyle          : "com.puppycrawl.tools:checkstyle:${pluginVersion.checkstyle}",
-            spotbugs            : "gradle.plugin.com.github.spotbugs:gradlePlugin:${pluginVersion.spotbugs}",
-            coveralls           : "org.kt3k.gradle.plugin:coveralls-gradle-plugin:${pluginVersion.coveralls}",
-            errorprone          : "net.ltgt.gradle:gradle-errorprone-plugin:${pluginVersion.errorprone}",
-            dependencyGraph     : "com.vanniktech:gradle-dependency-graph-generator-plugin:${pluginVersion.dependencyGraph}",
-            dependencyUpdates   : "com.github.ben-manes:gradle-versions-plugin:${pluginVersion.dependencyUpdates}",
-            kotlinGradle        : "org.jetbrains.kotlin:kotlin-gradle-plugin:${androidVersions.kotlinVersion}",
+            gradle             : "com.android.tools.build:gradle:${pluginVersion.gradle}",
+            checkstyle         : "com.puppycrawl.tools:checkstyle:${pluginVersion.checkstyle}",
+            spotbugs           : "gradle.plugin.com.github.spotbugs:gradlePlugin:${pluginVersion.spotbugs}",
+            coveralls          : "org.kt3k.gradle.plugin:coveralls-gradle-plugin:${pluginVersion.coveralls}",
+            errorprone         : "net.ltgt.gradle:gradle-errorprone-plugin:${pluginVersion.errorprone}",
+            dependencyGraph    : "com.vanniktech:gradle-dependency-graph-generator-plugin:${pluginVersion.dependencyGraph}",
+            dependencyUpdates  : "com.github.ben-manes:gradle-versions-plugin:${pluginVersion.dependencyUpdates}",
+            kotlinGradle       : "org.jetbrains.kotlin:kotlin-gradle-plugin:${androidVersions.kotlinVersion}",
+            kotlinSerialization: "org.jetbrains.kotlin:kotlin-serialization:${androidVersions.kotlinVersion}",
```

Comment:
Why do we need kotlinx and kotlin serialization?

Reply:
I don't understand the questions.

The Gradle-Kotlin is for get Kotlin run (was added already before). And the serialization one is for having annotations processing of for JSON parsing.

Reply:
Or do you mean, why we need `kotlinSerialization ` **and** `kotlinxSerializationJson`?

Don't look at the map-keys. One of them is the library, and the other one the annotation processing plugin.

But I think I should adjust the naming.

Reply:
Ah ok my bad, I think I was just wondering about the naming.